docs/content/operations/api-reference.md

```diff
@@ -230,6 +230,20 @@ Enables all segments of datasource which are not overshadowed by others.
 
 Enables a segment of a datasource.
 
+##### PUT
+
+* `/druid/coordinator/v1/datasources/{dataSourceName}/markUsed`
+
+Enables segments of a datasource by the interval or by an array of segment id's passed in the request payload. 
+It will enable only those segments which are not overshadowed by other sergments.
+
+JSON Payload:
+
+|Key|Description|Example|
+|----------|-------------|---------|
+|`interval`|Interval of segments to enable|"2015-09-12T03:00:00.000Z/2015-09-12T05:00:00.000Z"|
```

Comment:
The docs need to call out what happens when both are set, and the inclusive/exclusive behavior for the intervals